tutorials/mie_potential/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "mie_potential"
+version = "0.1.0"
```

Comment:
I would use version = "0.0.0" here, and add a `publish = false` to ensure this is never published to crates.io